src/String.js

```diff
@@ -95,9 +95,12 @@ function encodingWidth(encoding) {
   switch(encoding) {
     case 'ascii':
     case 'utf8': // utf8 is a byte-based encoding for zero-term string
+    case 'x-mac-roman':
```

Comment:
fontkit uses a few others as well. Need fallback handling? Or is it an API usage issue?

Reply:
As this issue can cause an infinite loop, I would expect either:
- a fallback
- an explicit exception if the value is not handled and will possibly cause an infinite loop

Reply:
I don't think there's an infinite loop here. Perhaps the calling code or its callers have a loop?

Reply:
@renchap for now, I have it assume 1-byte, which matches previous behavior